src/lib/components/EventNetwork.svelte

```diff
@@ -385,11 +554,28 @@
 </div>
 
 <style>
-  :global(.network-link-leather) {
-    @apply stroke-gray-400 dark:stroke-gray-600 fill-gray-400 dark:fill-gray-600;
+  .legend-list {
+    @apply list-disc pl-5 space-y-2 text-gray-800 dark:text-gray-300;
+  }
+
+  .legend-item {
+    @apply flex items-center;
+  }
+
+  .legend-icon {
+    @apply relative w-6 h-6 mr-2;
+  }
+
+  .legend-circle {
+    @apply absolute inset-0 rounded-full border-2 border-black;
+  }
+
+  .legend-circle.content {
+    @apply bg-gray-700 dark:bg-gray-300;
+    background-color: #d6c1a8;
```

Comment:
It would be more idiomatic for Tailwind to put some of these utility classes directly onto the elements they are meant to style, especially when a set of styles is used only once.

So `.legend-list` should be removed, and the utility classes applied directly to the element inline.

The other classes are reused, so it makes more sense to define those here in the style section.